definition/tangles.js

```diff
@@ -0,0 +1,39 @@
+module.exports = {
```

Comment:
:fire: I don't think this file is needed
This module is mainly made to offer tools for building common schema. Right not `tangles.profile` isn't used enough for us to safely put this in here.